init/init.go

```diff
@@ -349,7 +349,6 @@ func RunInit() error {
 		},
 		createGroups,
 		extractModules,
-		loadModules,
```

Comment:
We can't remove this here.  We need this incase you need a module to mount the disk.  We can just have it twice.  Just make sure if you load a module twice it doesn't fail.